commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt

```diff
@@ -3,6 +3,7 @@ package ebi.ac.uk.api.security
 import com.fasterxml.jackson.annotation.JsonProperty
 import javax.validation.constraints.Email
 
+@Suppress("LongParameterList")
```

Comment:
I see a lot of these were added during this refactor. Shouldn't we just adjust the parameter to something that makes more sense to us?

Reply:
lets discuss this and address in individual version